bindings/java/Swig/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.cntk</groupId>
+    <artifactId>cntk</artifactId>
+    <version>2.4</version>
```

Comment:
What is the impact of this version? Can it be a dynamic string? Version 2.4 won't comply with our nightly builds. Our nighlty builds has format 2.5-rc0.dev{yyyyMMdd}. You can check how I added dynamic version support in Python WHL creation (bindings/python/setup.py)

Reply:
unfortunately its xml so there is no easy way to add dynamic support from within the xml
however this only gets used when you actually publish the jar and is not used in the build process

Reply:
Great then.